src/tensor_ops/map.rs

```diff
@@ -35,6 +35,46 @@ pub fn relu<T: Tensor<Dtype = f32>>(t: T) -> T {
     map_df_uses_fx(t, |x| x.max(0.0), |fx| if fx > &0.0 { 1.0 } else { 0.0 })
 }
 
+/// [Leaky Rectified Linear Unit (Leaky ReLU)](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Leaky_ReLU). `max(0.01 * t, t)`
+///
+/// The derivative is '1', for `t >= 0`, and `0.01` elsewhere.
+///
+/// Examples:
+/// ```rust
+/// # use dfdx::prelude::*;
+/// let t = tensor([-1.0, 0.0, 1.0, 2.0]);
+///
+/// // use function version
+/// let r = leakyrelu(t.clone());
+///
+/// // or the tensor method!
+/// let r2 = t.leakyrelu();
+/// ```
+pub fn leakyrelu<T: Tensor<Dtype = f32>>(t: T) -> T {
+    map_df_uses_fx(t, |x| x.max(0.01 * x), |fx| if fx > &0.0 { 1.0 } else { 0.01 })
+}
+
+/// [Softplus](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Softplus). `ln(1 + e^t)`
+///
+/// The derivative is the [Logistic](https://en.wikipedia.org/wiki/Logistic_function) function. `1 / (1 + e^-t)`
+///
+/// Examples:
+/// ```rust
+/// # use dfdx::prelude::*;
+/// let t = tensor([-1.0, 0.0, 1.0, 2.0]);
+///
+/// // use function version
+/// let r = softplus(t.clone());
+///
+/// // or the tensor method!
+/// let r2 = t.softplus();
+/// ```
+pub fn softplus<T: Tensor<Dtype = f32>>(t: T) -> T {
+    map_df_uses_fx(t,
```

Comment:
Softplus derivative should just use normal x instead of f(x)
```suggestion
    map(t,
```